js/apps/admin-ui/src/components/time-selector/TimeSelector.tsx

```diff
@@ -88,9 +92,11 @@ export const TimeSelector = ({
     if (value) {
       setMultiplier(multiplier);
       setTimeValue(value / multiplier);
+      setLastMultiplier(multiplier);
     } else {
       setTimeValue(value || "");
-      setMultiplier(defaultMultiplier);
+      setMultiplier(lastMultiplier || defaultMultiplier);
+      setLastMultiplier(lastMultiplier || defaultMultiplier);
```

Comment:
Note that `||` will also fall back to `defaultMultiplier` if `lastMultiplier` is `0`. This is probably intentional, as that is an invalid multiplier, but if you'd like to only fall back if `lastMultiplier` is `undefined` you can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) (`??`).

```suggestion
      setMultiplier(lastMultiplier ?? defaultMultiplier);
      setLastMultiplier(lastMultiplier ?? defaultMultiplier);
```

Again, not sure if this suggestion applies here, feel free to do what you like.